### ISSUE TITLE
Create a GitHub issue if compatibility breaks

### ISSUE THREAD

**Issue by SeanNaren** [PyTorchLightning/ecosystem-ci#30]
## 🚀 Feature

It would be awesome if we could set the CI up to make an issue on the corresponding CI if something was to break. 

Even better would be to have a stack trace showing what broke! Something like below:

Hi! I'm the [PyTorchLightning EcoSystem-CI](https://github.com/PyTorchLightning/ecosystem-ci) bot.

I've detected an incompatibility with your repository and Lightning's release (1.5.x). I've attached the stack trace below for help in debugging.

<details open>
<summary>Stack Trace (24/01/22)</summary>
<br>
<STACKTRACE
</details>

Any additional breakages could update the same issue!

**Comment by Borda** [comment]
we can you can create an issue about it but a challenge would be balancing how many new issues are created if the previous was not resolved... imagine having new issue every 24h just because you have not reacted yet

well there would be a smart way to create a new of none related EcoCI exists and if there is just comment on existing issue

**Comment by SeanNaren** [comment]
Yes thank you @Borda :) I think it would be great that it would make one issue, and somehow pile all the failures onto the same issue.

Maybe like as suggested, it is able to add the stack trace in drop downs, with the date corresponding to the failure. At the end of the day, we only care about the latest one (nightly I assume).

**Comment by Borda** [comment]
let's explore this API: https://github.com/fastai/ghapi :rabbit: thx @aribornstein
